.github/workflows/create-release.yml

```
@@ -1,4 +1,4 @@
-name: Automated releases
+name: Create Release PRs
```

Since the `add-labels` workflow is named using "Pull Request" instead of the acronym, does not make sense use that here also? Additionaly, shouldn't the name be singular ("Pull Request" instead of "Pull Requests" or "PRs")?

```suggestion
name: Create Release Pull Request
```

---

Good suggestion, I agree. Writing it in full I feel like singular makes more sense.